### PR TITLE
Overview topic

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) creates an overview topic for Commerce Eventing

Preview build: https://adobedocs.github.io/commerce-extensibility/events/

## Affected pages

-  ...

## Links to related PRs or Jira tickets

-  ...

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My changes follow the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/events/index.md [hunk 1]
```diff
@@ -1,85 +1,18 @@
...
-or deletions for this documentation, check out the source from [this github repo](https://github.com/adobe/gatsby-theme-spectrum-example), and submit a pull
-request with your contribution.
-
-## API Requests & Rate Limits
-
-The timeout for API requests through adobe.io is currently *60 seconds*.
-
-The default rate limit for an Adobe Analytics Company is *120 requests per minute*. (The limit is enforced as *12 requests every 6 seconds*).
-When rate limiting is being enforced you will get `429` HTTP response codes with the following response body: `{"error_code":"429050","message":"Too many requests"}`
+This architecture helps merchants efficiently customize processes and integrate systems while maintaining SaaS-like upgradeability. Before event-driven applications, developers were often required to use APIs to poll to determine if something had occurred. Depending on polling frequency, there could be lags. As a result the merchant could be missing out on realtime activities. Shoppers might not have the best possible experiences.
```
Comment (jhadobe) [inline src/pages/events/index.md:18]:
```suggestion
This architecture helps merchants efficiently customize processes and integrate systems while maintaining SaaS-like upgradeability. Before event-driven applications, developers had to use API polling to determine if something had occurred. Lags in polling could result in merchants missing realtime activities, which could prevent shoppers from having an optimal experience.
```

File: src/pages/index.md [hunk 2]
```diff
@@ -1,85 +1,18 @@
...
+![Commerce Extensibility](_images/home-bg.jpeg)
 
-Adobe Product API offers limitless ways to integrate your most important customer data into key business processes. Adobe Product API offer limitless ways.
+# Adobe Commerce Extensibility
 
-<Resources slots="heading, links"/>
+Learn how to use the Adobe infrastructure to create extensions for Adobe Commerce and Magento Open Source.
 
-#### Resources
+*  [App Builder](https://developer.adobe.com/app-builder/docs/overview/) is a complete framework that enables enterprise developers to build and deploy custom web applications that extend Adobe Experience Cloud solutions and run on Adobe infrastructure.
```
Comment (jhadobe) [inline src/pages/index.md:14]:
```suggestion
*  [App Builder](https://developer.adobe.com/app-builder/docs/overview/) is a complete framework that allows enterprise developers to build and deploy custom web applications that extend Adobe Experience Cloud solutions and run on Adobe infrastructure.
```

File: src/pages/index.md [hunk 3]
```diff
@@ -1,85 +1,18 @@
...
 
-* [Quickstart Guide](https://www.adobe.io/apis/experiencecloud/analytics/docs.html)
-* [Adobe Analytics Github Repo](https://github.com/AdobeDocs/analytics-2.0-apis)
+*  [Adobe I/O Events for Adobe Commerce](./events/index.md) makes all Commerce transactional data available to App Builder using Adobe I/O. Once the data is available in I/O, developers can use App Builder to build extensions and run custom integrations with 3rd party applications.
```
Comment (jhadobe) [inline src/pages/index.md:16]:
```suggestion
*  [Adobe I/O Events for Adobe Commerce](./events/index.md) makes all Commerce transactional data available to App Builder using Adobe I/O. Once the data is available in I/O, developers can use App Builder to build extensions and run custom integrations with third-party applications.
```

**Review by jhadobe — CHANGES_REQUESTED** [review]

File: src/pages/index.md (hunk 2 above)
Comment (keharper) [reply src/pages/index.md:14]: I'm going to keep this one as is. We're empowering developers, not giving them permission.

**Review by jhadobe — APPROVED** [review]

**Review by macytd — APPROVED** [review]

**Pull request merged**